manifests/init.pp

```diff
@@ -104,13 +110,24 @@
   Array[String] $repo_exclusions = [],
   Hash[String, Hash[String, String]] $gpgkeys = {},
   String $utils_package_name = 'yum-utils',
+  Boolean $purge_unmanaged_repos = false,
+  Stdlib::Unixpath $repodir,
```

Comment:
please set the default value here, not in hiera. This enables puppet-strings to pick up the value for the REFERENCE.md

Reply:
Done